### ISSUE TITLE
Confused by MacOS installation instructions

### ISSUE THREAD

**Issue by mroberts1** [MasterQ32/kristall#60]
I am trying to install Kristall on MacOS/Catalina, but the instructions are pretty cryptic:

"Use the Makefile to build build/kristall instead of the default target. There is currently no auto-recognition of MacOS (where qmake will output a kristall.app file), so you have to be a bit more manual."

I cloned the Kristall repo and know how to edit Makefile. I also installed qt5 and qt-editor via Homebrew. But now I am stuck - apparently I need to install  "widgets and network modules," which I have no idea how to do, while it isn't at all clear how I have to edit the Makefile.

Is this app only for experienced qt users? If not, I'd be grateful for some more helpful instructions.

**Comment by MasterQ32** [comment]
> Is this app only for experienced qt users? If not, I'd be grateful for some more helpful instructions.

Yeah, at the moment for sure. I never developed anything on MacOS, but people already built it successful on Mac, but those were experienced devs that had used Qt already.

Maybe someone with Mac experience can help you here

**Comment by mroberts1** [comment]
Ok no worries!

**Comment by ljcooke** [comment]
I'm not familiar with Qt, but these steps worked for me on macOS 10.14/Mojave:

```sh
brew install qt
make build/kristall
open build/kristall.app
```

As far as I can tell, the Qt install with Homebrew includes the widgets and network modules.

**Comment by jahzielv** [comment]
I'm also having trouble building on Mac. I ran `make build/kristall` after installing qt, but I get an error related to openssl:
![image](https://user-images.githubusercontent.com/19838370/92313137-1bdfb500-ef96-11ea-8960-6f8123fdc0b5.png)

I've checked and the openssl headers are symlinked into `/usr/local/include`, which should mean that gcc can find them, but it's not for some reason. Has anyone had this problem? I can also open a new issue if need be, since this problem is only slightly related to the one in this issue.
